modules/nf-core/parabricks/fq2bam/main.nf

```diff
@@ -38,12 +40,12 @@ process PARABRICKS_FQ2BAM {
     # index and fasta need to be in the same dir as regular files (not symlinks)
     #   and have the same base name for pbrun to function
     #   here we copy the index into the staging dir of fasta
-    FASTA_PATH=`readlink -f $fasta`
-    cp \$INDEX.amb \$FASTA_PATH.amb
-    cp \$INDEX.ann \$FASTA_PATH.ann
-    cp \$INDEX.bwt \$FASTA_PATH.bwt
-    cp \$INDEX.pac \$FASTA_PATH.pac
-    cp \$INDEX.sa \$FASTA_PATH.sa
+    # FASTA_PATH=`readlink -f $fasta`
```

Comment:
Hmm ... just trying to understand this 🤔

The old code just copied the index-files right?  By adding the directive `stageInMode "copy"` you are also copying the input fastq-files and fasta-file?